packages/flutter/lib/src/material/toggle_buttons.dart

```diff
@@ -1550,3 +1572,138 @@ class _SelectToggleButtonRenderObject extends RenderShiftedBox {
     }
   }
 }
+
+/// A widget to pad the area around a [ToggleButtons]'s children.
+///
+/// This widget is based on a similar one using in [ButtonStyleButton] but it
```

Comment:
there's an issue with this sentence. Overall I don't understand this comment. What does "tap target" in quotes mean? Which widget does "size of the widget" refer to?

Reply:
Reworded the sentence, hopefully thats better.

Reply:
```suggestion
/// This widget is based on a similar one used in [ButtonStyleButton] but it
```

Reply:
Yes

Reply:
oops, fixed